src/utilities/getIntrospectionQuery.js

```diff
@@ -58,6 +58,7 @@ export function getIntrospectionQuery(options?: IntrospectionOptions): string {
       kind
       name
       ${descriptions}
+      specifiedByUrl
```

Comment:
@m14t Since older GraphQL servers don't know about this field it will result in the entire query being rejected.
I'm actively working on a two-stage introspection solution that is a generic solution to this problem.
But in the mean time, you need to flag similar one for schema description:
https://github.com/graphql/graphql-js/pull/2419/files#diff-72ccdfc3a6c358028739a30a14edddce
Also please add tests:
https://github.com/graphql/graphql-js/pull/2419/files#diff-80e66acfef544cd667aab53b088cc9ed

Reply:
Done in 18689db4cd0cb9c031ea7054b55da3ebe796e034.

Should this default to `true` or `false`?  This commit defaults it to `true`.

Reply:
@m14t `false` otherwise it would be a breaking change.

Reply:
Done in 4dcf49be.

Reply:
Apologies, changing the default from `true` to `false` required passing this argument as `true` in two other tests that was missed.  Fixed in 17d4ad38.